internal/mode/static/manager.go

```diff
@@ -149,9 +150,11 @@ func StartManager(cfg config.Config) error {
 	processHandler := ngxruntime.NewProcessHandlerImpl(os.ReadFile, os.Stat)
 
 	// Ensure NGINX is running before registering metrics & starting the manager.
-	if _, err := processHandler.FindMainProcess(ctx, ngxruntime.PidFileTimeout); err != nil {
+	p, err := processHandler.FindMainProcess(ctx, ngxruntime.PidFileTimeout)
+	if err != nil {
 		return fmt.Errorf("NGINX is not running: %w", err)
 	}
+	cfg.Logger.Info("NGINX is running with PID", "pid", p)
```

Comment:
Let's log this at the debug level

Reply:
I wanted to do that...but I couldn't find a `.Debug`?

Reply:
There's no debug method, you have to use `.V`:

`logger.V(1).Info()`